js/site-orientation.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+var $ = require('jquery');
+var countdown = require('countdown');
+var introJs = require('intro.js');
+var URI = require('urijs');
+
+var helpers = require('./helpers');
+
+var uri = window.location.toString();
+var uriQuery = helpers.sanitizeQueryParams(URI.parseQuery(window.location.search));
+
+function SiteOrientation(selector) {
+  this.$selector = $(selector);
+
+  this.$banner = this.$selector.find('.banner');
+  this.$bannerToggleSection = this.$banner.find('.toggle');
+  this.$bannerToggleLink = this.$banner.find('.toggle-text');
+  this.$bannerMoreText = this.$bannerToggleLink.find('.more');
+  this.$bannerLessText = this.$bannerToggleLink.find('.less');
+
+  this.$tourHeader = this.$selector.find('.tour-header');
+
+  this.$startTourLink = this.$selector.find('.start-tour');
+  this.$startTourLink.on('click', this.startTour.bind(this));
+
+  if (uriQuery.tour) {
+    this.startTour();
+  }
+  else {
+    this.initBanner();
+  }
+}
+
+SiteOrientation.prototype.initBanner = function () {
+  this.$banner.show();
+
+  // fill in _ DAYS until this site...
+  this.$days = this.$selector.find('.days');
+  var countdownDate = new Date(this.$days.data('date'));
+  this.$days.html(countdown(countdownDate, null, countdown.DAYS).days);
+
+  // anonymous feedback tool click
+  this.$selector.on('click', '.js-feedback', function () {
+    $(document.body).trigger('feedback:open');
+  });
+
+  if (localStorage.getItem('FEC_BANNER_COLLAPSED') === 'true') {
+    this.collapseBanner();
+  }
+
+  // unbind to prevent multiple click actions
+  this.$bannerToggleLink.unbind().on('click', this.handleToggle.bind(this));
+};
+
+SiteOrientation.prototype.handleToggle = function () {
+  this.$bannerToggleSection.toggle();
+  this.$bannerLessText.toggle();
+  this.$bannerMoreText.toggle();
+
+  this.setBannerState();
+};
+
+SiteOrientation.prototype.collapseBanner = function () {
+  this.$bannerToggleSection.hide();
+  this.$bannerLessText.hide();
+  this.$bannerMoreText.show();
+};
+
+SiteOrientation.prototype.setBannerState = function () {
+  if (this.$selector.find('.more').is(':visible')) {
+    localStorage.setItem('FEC_BANNER_COLLAPSED', 'true');
+  }
+  else {
+    localStorage.setItem('FEC_BANNER_COLLAPSED', 'false');
+  }
+};
+
+SiteOrientation.prototype.startTour = function () {
+  var self = this;
+
+  // if the user clicks "start a tour" on a page without tooltips
+  // then it will take them to the homepage and start the tour
+  if (typeof TOUR_PAGE == 'undefined') {
+    window.location.href = CMS_URL + '/?tour=true';
```

Comment:
It's declared in many places on webapp, and it looks like footer and nav uses it as well. Before next release we should just double check to make sure it's set on both.